build.gradle.kts

```diff
@@ -1,3 +1,15 @@
+buildscript {
+    repositories {
+        gradlePluginPortal()
+        google()
+        mavenCentral()
+    }
+    dependencies {
+        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:1.6.21")
```

Comment:
How about setting up a `buildSrc` module to manage these dependencies?

Reply:
I missed this one, let me implement it on another chore task after #67 is merged 🙏